### PR TITLE
add unit tests widget

### PR DESCRIPTION
**Pull request by qlqllu**
This is the sample widget will be used in the UT doc: https://github.com/ArcGIS/afd-experience-builder-doc/pull/328

### CODE REVIEW

**Comment by qlqllu** [comment]
@gavinr @shawnmgoulet 
This PR is ready, can you help review?

**Comment by qlqllu** [comment]
@gavinr @shawnmgoulet This PR is ready for review. The error when run UT has been fixed but the fix will be get merged tomorrow and will be available in next build.

**Comment by shawnmgoulet** [comment]
> @gavinr @shawnmgoulet This PR is ready for review. The error when run UT has been fixed but the fix will be get merged tomorrow and will be available in next build.

thanks @qlqllu. You may have this on your mind, but just want to point out we suggested changing the name of the widget to `show-unit-tests` rather than abbreviating. Will hold until tomorrow (build 646) to test.

**Comment by qlqllu** [comment]
@shawnmgoulet The widget is renamed.

File: widgets/show-unit-tests/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "show-ut",
```
[the remark this answers is not in the document]
Comment (shawnmgoulet) [reply widgets/show-unit-tests/manifest.json]: @gavinr @qlqllu https://github.com/Esri/arcgis-experience-builder-sdk-resources/pull/81/commits/d514301a323bca26a5a265d33f7dab37122c39dd should take care of this.

**Comment by shawnmgoulet** [comment]
@qlqllu https://github.com/Esri/arcgis-experience-builder-sdk-resources/commit/d514301a323bca26a5a265d33f7dab37122c39dd should address my changes. I ran locally and it returned below, which I believe is expected without adding any widgets to an experience.

```
$ npm run test

> exb-client@1.5.0 test C:\Projects\SoftwareDevelopment\ExperienceBuilderDeveloperEnablement\1-5\649\arcgis-experience-builder\client
> jest --runInBand

ts-jest[config] (WARN) The option `tsConfig` is deprecated and will be removed in ts-jest 27, use `tsconfig` instead
 PASS  your-extensions/widgets/show-unit-tests/tests/widget.test.tsx (5.856 s) PASS  your-extensions/widgets/simple/tests/simple-widget.test.tsx

Test Suites: 2 passed, 2 total
Tests:       6 passed, 6 total
Snapshots:   0 total
Time:        8.235 s
```

**Review by shawnmgoulet — APPROVED** [review]
https://github.com/Esri/arcgis-experience-builder-sdk-resources/commit/d514301a323bca26a5a265d33f7dab37122c39dd incorporates my suggestions (updating name in additional files). Tested the widget and it returned unit test results.

**Pull request merged**